scss/partials/_job.scss

```diff
@@ -0,0 +1,82 @@
+// VT Jobs cards, details, and search
+.job-card {
+    @extend .content-block--clickable;
+    border: 0.2rem solid $secondary-green-2;
+    display: block;
+    height: 16.5rem;
+    width: 40rem;
+
+    @include breakpoint(desktop) {
+        width: 56rem;
+    }
+
+    &__block {
+        padding: 2.0rem 1.3rem 2.1rem;
+    }
+
+    &__title {
+        @extend .h6;
+    }
+
+    &__location {
+        @extend .tk-aktiv-grotesk;
+        @extend .uppercase;
+        color: $secondary-green-1;
+        font-size: 1.2rem;
+        letter-spacing: 1.8px;
+    }
+
+    &__posted {
+        @include self() {
+            @extend .tk-degular-text;
+            display: block;
+            font-size: 1.6rem;
+            font-weight: 500;
+            padding-top: 1.5rem;
+
+            @include breakpoint(desktop) {
+                padding-top: 4.5rem;
+            }
+        }
+
+
+        &--new {
+            &::before {
+                background-color: $secondary-orange-2;
+                border: 0.1rem solid $secondary-orange-2;
+                color: $secondary-blue-1;
+                content: "NEW";
+                display: inline-block;
+                font-size: 1.4rem;
+                font-weight: 700;
+                letter-spacing: 3.2px;
+                padding-left: 0.5rem;
+                margin-right: 1rem;
+            }
+        }
+    }
+
+    &:hover,
+    &:focus {
+        background-color: $primary-blue;
+        border: 0.2rem solid $primary-blue;
+        transition: background-color 0.2s ease-out;
+
```

Comment:
Couldn't think of a better solution to make all the elements change color, transition, and border color on hover.